.github/workflows/go.yml

```diff
@@ -11,29 +11,28 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: [1.16.x, 1.17.x, 1.18.0-beta1]
+        race: ["-race", ""]
+
     steps:
 
-    - name: Set up Go 1.x
+    - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: ^1.13
+        stable: '!contains(${{ matrix.go-version }}, "beta") && !contains(${{ matrix.go-version }}, "rc")'
+        go-version: ${{ matrix.go-version }}
 
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
-      
+
     - name: Install dbus
       run: sudo apt-get install dbus dbus-x11
-      
-    - name: Get dependencies
-      run: |
-        go get -v -t -d ./...
-        if [ -f Gopkg.toml ]; then
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            dep ensure
-        fi
 
     - name: Build
-      run: go build -v ./...
+      run: go build "${{ matrix.race }}" -v ./...
```

Comment:
I think you need to omit the quotes here

Reply:
right